### ISSUE TITLE
Synchronization problems on Tecno tablets with Android 7

### ISSUE THREAD

**Issue by bernardsilenou** [hzi-braunschweig/SORMAS-Project#3934]
### Bug Description
This are just summary points after testing this version of the mobile app. We would need to further reproduce them and translate ti separate issues if needed.

- Techno
- Menu language of app ie captions and enums ere not translated after instalation.
- does not translate the interface languages at initial installation. After creating a case, synk with the server, edit the case from web and and 
sync back to tablet, the languages was changed to English. Setting language back to German and when to English maintains the language in German.
Thus language settings are not stable

- Default language of app is German, It is better we change to English.

- During case creation, when app is left attended for about 5 secs or even when actively entering data, all previously entered data on the form vanishes.
This is not related to the time to sleep of the tablet because this happens when the tablet has not gone to sleep yet.
On sansung and huawei (Adroid version 7.0), this does not happen. Even when the samsung goes to sleep and latter turn on, data is not deleted and user van still continue entering data.

### Steps to Reproduce
1. Get a new apk and install on the Techno tablet used in Nigeria
2. Observer language of the app
3. create a new case and observer how it works

### System Details
* Device: Tecno, android version 7.0
* SORMAS version: 1.53.0
* Android version/Browser: tablet
* Server URL: https://release-international.sormas.netzlink.com/sormas-ui/
* User Role: informant

**Comment by bernardsilenou** [comment]
@MateStrysewske Please can you have a second look into this?
What works fine:
  - Automatic update
  - bi-directional sync with server
  - editing of entities after creation was ok

**Comment by bernardsilenou** [comment]
I initially though this was due to the fact that the android is 7, but huawei is also on 7. I would upgrade to 8, test again and comment.

**Comment by bernardsilenou** [comment]
@MateStrysewske 
I upgraded the Tecno to android version 7, build model P701-PB04A-N-170523V26, install a new apk and all works perfect.
Maybe the older model had an issue.
The memory of the tablet is limited and could not download this new model. I had to us an SD card to store the download before installing it.

@Chinedar @foyiri @Iheanacho2027 I got informed that the mobile users are facing synchronization issues. I am beginning to suspect that this may not be a connectivity issue but that they may not have the right model of android version.
Please can you check that all the users have the right model?

**Comment by Iheanacho2027** [comment]
@bernardsilenou I doubt this is an issue of the mobile device or android version being used, because users with Samsung tab A 8.0" and 10.1" (Android 9.0 ) are all having synchronization issues. Issues ranging from error messages ("can't connect to SORMAS server") to long periods taken for a successful synchronization to occur. This happens with good internet connection. It's important we have a fix to this

**Comment by bernardsilenou** [comment]
@Iheanacho2027 Good to know that. Please quickly visit some of the users with connectivity issues and collect all the data we discussed about so that we can diagnose the issue.
- You can take different types of devices, login as a second DSNO and collect data on the performance
- Another thing to remove entities on the mobile devices, the devices may be overloaded as explained below:

**Comment by bernardsilenou** [comment]
- @MateStrysewske I just realized archived entities are not deleted from the mobile device after sync with db. Don't you think this can affect the performance of the devices?
- Deleted cases also stay on mobile device. I "RE-SYNC-DATA" hopping that this is caused by cach and got error message "synchronization failed please try again latter" but deleted case was gone from tablet.
- I then archives anther case, synch and it was not gone, re-sync and got same error message, open case listing and case was gone. re-sync works but returns error message, don't know why
- In summary: entities are only deleted from tablet after a re-sync + being archived/ deleted from db. I would suggest that we clear the cach of the tablet and also delete all entities in the tablet that are either archived on deleted from server end. What do you think please?

**Comment by MateStrysewske** [comment]
@bernardsilenou Deleted and archived entities are removed from the mobile apps once per day. Checking for deleted or archived entities with every normal synchronisation would slow down the process too much.

**Comment by bernardsilenou** [comment]
@MateStrysewske thanks for this vital information. I think the current implementation per day is fine

**Comment by bernardsilenou** [comment]
@MateStrysewske More comments:
- Manually upsetting app using apk did installed a new version and did not log the user out. This is very good because in case auto update does not work for some reason, than we can send the apk to users to upgrade manually.
- I installed a new app 1.53 on on a Tecto tablet with a different build number as shown in figure below and the issues that data get lost persisted.
![image](https://user-images.githubusercontent.com/12996724/104062401-b4af6700-51fa-11eb-8adb-22da1f1b830f.png)

- Next diagnostic was to upgrade to android model P701-PB04A-N-170523V26 and test again. I tried but tablet could no upgrade.
 In summary, the Tablets with versions lesser than P701-PB04A-N-170523V26 would nto be able to use this new version.

- @Chinedar @foyiri @Iheanacho2027  Are there still tablets having android < 8 in the field ? If yes then we have to ansolutely replace them.

**Comment by Chinedar** [comment]
Yes @Silenou, Bernard <Bernard.Silenou@helmholtz-hzi.de> we have begun
replacing them. However, is there a way to update the application to be
compatible with android 7? We stand to lose a lot of data yet to
synchronize.

**Comment by bernardsilenou** [comment]
@MateStrysewske  Further comments on testing with Tecno on android 7, P701-PB04A-N-170523V26:
  * tablet was working and old user (informant) was logged out and another user (SO) registered.
   - login was successful but app could not sync with server.  user account logged  good. All directories were empty.
   - The same problem did not occur on Huawei android 7.0, model BG2-U01C199B006 
   
Please can you have a second look at this? Initial setup was fine and when a new user was logged on the tablet, nothing works.

**Comment by MateStrysewske** [comment]
@Chinedar @bernardsilenou The application supports Android 7, we're also testing it on that version. Bernard's comment also suggests that it works on another model with Android 7. I'll try to test the app on a Tecno tablet and see whether I can find out what the issue is, but that's unfortunately not possible for the upcoming release.

**Comment by MateStrysewske** [comment]
@bernardsilenou Is this still a problem?

**Comment by bernardsilenou** [comment]
I would close this issue  since much development has taken place over time.
